### ISSUE TITLE
When entering the Appearance menu, font choice is reset to Source Code Pro

### ISSUE THREAD

**Issue by toph-allen** [blinksh/blink#701]
When entering the Appearance menu, the font selection is reset to Source Code Pro. Correspondingly, after entering and exiting that menu, even without selecting a new font, all subsequent shells opened will use Source Code Pro.

The attached video shows the bug in action. Entering and exiting the Appearance menu, and then hitting Cmd+W to close the current shell, causes the next shell to use Source Code Pro.

[IMG_0773_TRIM.zip](https://github.com/blinksh/blink/files/2971212/IMG_0773_TRIM.zip)

**Comment by yury** [comment]
Hi @toph-allen,

Thanks! Will be fixed in next TF release.

**Comment by toph-allen** [comment]
Confirming, this is resolved in Build 93.

**Comment by yury** [comment]
Thanks, closing then.

**Comment by toph-allen** [comment]
I noticed this issue reoccur under Build 93 #702. After some experimentation, it seems like it happens only with fonts which I installed myself using [AnyFont](https://itunes.apple.com/us/app/anyfont/id821560738?mt=8)/[FondFont](https://itunes.apple.com/us/app/fondfont-install-system-fonts/id842109099?mt=8). I'm sure that installing non-standard fonts via provisioning profiles is really an edge case, though.

Perhaps, though, this offers some insight into #702? Maybe there's an issue with the way that (1) loading preferences and (2) computing width occurs for fonts which a user installed via a provisioning profile.
